zombies.js

```diff
@@ -193,6 +282,18 @@
  * @param {Item/Weapon/Food} item   The item to use.
  */
 
+Player.prototype.useItem = function (item) {
+  if (item instanceof Weapon && this.getPack().includes(item)) {
+    this.getPack().splice(this.getPack().indexOf(item), 1);
```

Comment:
You should use the `eat` and `equip` methods you created earlier here instead of having duplicate code.